ccmlib/cmds/cluster_cmds.py

```diff
@@ -524,9 +524,12 @@ def get_parser(self):
         parser.add_option('-v', '--verbose', action="store_true", dest="verbose",
                           help="Print standard output of cassandra process", default=False)
         parser.add_option('--no-wait', action="store_true", dest="no_wait",
-                          help="Do not wait for cassandra node to be ready", default=False)
-        parser.add_option('--wait-other-notice', action="store_true", dest="wait_other_notice",
-                          help="Wait until all other live nodes of the cluster have marked this node UP", default=False)
+                          help="Do not wait for cassandra node to be ready. Overrides all other wait options.", default=False)
+        # This option (wait-other-notice) is now deprecated, as it was never respected
+        parser.add_option('--wait-other-notice', action="store_true", dest="deprecate",
+                          help="DEPRECATED/IGNORED: Use '--skip-wait-other-notice' instead. This is now on by default.", default=True)
+        parser.add_option('--skip-wait-other-notice', action="store_false", dest="wait_other_notice",
```

Comment:
Just noticed -- naming the destination of this flag `wait_other_notice` is very confusing.